en/components/navdrawer.md

```diff
@@ -17,92 +17,289 @@ _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI
 </div>
 <div class="divider--half"></div>
 
+### Dependencies
+To started with all needed dependencies you can use the `IgxNavigationDrawerModule` and import it in your application's `AppModule`:
+```typescript
+import { IgxNavigationDrawerModule } from 'igniteui-angular/main';
+```
+or
+```typescript
+import { IgxNavigationDrawerModule } from 'igniteui-angular/navigation-drawer';
+```
+And include it in the app module:
+```typescript
+@NgModule({
+    imports: [
+        IgxNavigationDrawerModule,
+        ...
+    ]
+})
+export class AppModule {
+}
+```
+
+> Alternatively both barrels export the `IgxNavigationDrawerComponent` and additional directives, so those can be declared/referenced separately if needed.
```

Comment:
What does "barrel" mean here?

Reply:
It's just a bundle module. Best description I can find: https://basarat.gitbooks.io/typescript/docs/tips/barrel.html
Could change the wording if it makes it harder to understand/translate.

Reply:
Thanks @damyanpetev.  I'll use the word "module".